Readme.md

```diff
@@ -69,6 +70,8 @@ bytes(1024 * 1.7, {decimalPlaces: 0});
 bytes(1024, {unitSeparator: ' '});
 // output: '1 KB'
 
+bytes(1024, {outputAsObject: true});
+// output: {value: '1',unit: 'KB'}
```

Comment:
I think Node would output a space after the `,` right?

Reply:
I'm seeing spaces around brackets and between values `{ value: '1', unit: 'KB' }` when I output to node.